it/src/main/java/org/corfudb/universe/universe/vm/VmUniverseParams.java

```diff
@@ -0,0 +1,81 @@
+package org.corfudb.universe.universe.vm;
+
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+import org.corfudb.universe.universe.Universe;
+import org.corfudb.universe.universe.UniverseParams;
+
+import java.time.Duration;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * Represents the parameters for constructing a VM {@link Universe}.
+ */
+@Getter
+@ToString(callSuper = true)
+public class VmUniverseParams extends UniverseParams {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)